Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -464,11 +477,21 @@ void TextureCacheBase::BindTextures()
 
 void TextureCacheBase::UnbindTextures()
 {
-  bound_textures.fill(nullptr);
+  for (size_t i = 0; i < bound_textures.size(); ++i)
```

Comment:
It's more that the call to `UnbindTextues` is absolutely pointless.

If the textures have changed, then the `Load` call replaces the bind. If the texture hasn't changed then the bind is bound with the same texture.

There is no way to actually end up with null in a bind point, except for `usedtextures`. Which is why I recommend passing `usedtextures` directly into `BindTextures` to gain the same effect.

Right now this PR changes the behavior slightly when `usedtextures` decreases and a bind slot is not invalidated.